tests/test_frame.py

```diff
@@ -4009,7 +4010,19 @@ def test_transpose() -> None:
     df = pd.DataFrame({"a": [1, 1, 2], "b": [4, 5, 6]})
     check(assert_type(df.transpose(), pd.DataFrame), pd.DataFrame)
     check(assert_type(df.transpose(None), pd.DataFrame), pd.DataFrame)
-    check(assert_type(df.transpose(copy=True), pd.DataFrame), pd.DataFrame)
+
+    msg = (
+        r"The copy keyword is deprecated and will be removed in a future version\. Copy"
```

Comment:
You can just include the beginning of the message, instead of the whole message.  E.g. `r"The copy keyword is deprecated"`